packages/botonic-react/tests/webchat/storage.test.jsx

```diff
@@ -56,6 +57,23 @@ describe('TEST: storage', () => {
     expect(sessionStorage.getItem('botonicState')).not.toBeNull()
   })
 
+  it('Stores botonicState in the memory storage', async () => {
```

Comment:
How about renaming the test to something like "botonicState can be stored in the in-memory storage and retrieved afterwards?